src/api/app/controllers/webui/requests_listing_controller.rb

```diff
@@ -26,15 +30,20 @@ def assign_attributes
     @url = requests_path
   end
 
-  def filter_requests
+  # apply filters that are based on constant values and not dynamically based on
+  # the available requests
+  def filter_requests_by_static_values
     params[:ids] = filter_by_involvement(@filter_involvement).ids
-    params[:creator] = @filter_creators if @filter_creators.present?
     params[:states] = @filter_state if @filter_state.present?
     params[:types] = @filter_action_type if @filter_action_type.present?
 
     @bs_requests = BsRequest::FindFor::Query.new(params, filter_by_text(params[:requests_search_text])).all
   end
 
+  def filter_creators
+    @bs_requests = @bs_requests.where(creator: [@filter_creators]) if @filter_creators.present?
```

Comment:
```suggestion
    @bs_requests = @bs_requests.where(creator: @filter_creators) if @filter_creators.present?
```
Square brackets where not needed before...

Reply:
True! Removed them again